### ISSUE TITLE
misleading answer to exercise "Exploring the Math Module"

### ISSUE THREAD

**Issue by wikfeldt** [swcarpentry/python-novice-gapminder#277]
In the answer to the "Exploring the Math Module" exercise in episode 6, where `math.pow()` and `math.sqrt()` are compared, it says:   
`... However, sqrt(x) doesn’t support negative arguments.`

`math.pow()` doesn't support negative arguments either (i.e. `math.pow(-1,0.5)`) gives a `ValueError`. So instead perhaps refer to the `cmath` library

**Comment by souravsingh** [comment]
'math.pow(-1, 2)' works without problem.

**Comment by wikfeldt** [comment]
yes absolutely, so the answer is not incorrect. Just a bit misleading (although maybe it's just me) because the discussion is about comparing sqrt and pow for taking square roots and the specific example `pow(x,0.5)` is brought up in the same context as negative arguments

**Comment by lo5an** [comment]
I just opened a PR removing the misleading sentence.  I believe I added this answer, and I was probably confusing `pow()` and `math.pow()` when I wrote it.
